### DISCUSSION TITLE
Scan Container Images from Multiple Container registries (ECR,AZURE,GCP) through CLI Flags

### DISCUSSION THREAD

**Discussion by Shivam-nagar23** [aquasecurity/trivy#4153] [Q&A] [answered]
I want to scan my images present in my private container registries through CLI Command flags (--username --password) . Does trivy supports scanning images through flags in ECR ,GCP and AZURE. 
I have Referred to the Docs, could not find anything related to flag in Private Container Registries.

**Comment by knqyf263** [comment, marked as answer]
You can reference it here.
https://aquasecurity.github.io/trivy/v0.41/docs/advanced/private-registries/
